examples/cmri_mpca/tutorial.ipynb

```diff
@@ -197,16 +199,21 @@
    },
    "outputs": [],
    "source": [
-    "# plot the first phase of images\n",
-    "visualize.plot_multi_images(\n",
-    "    [images[i][0, ...] for i in range(n_samples)],\n",
-    "    marker_locs=landmarks,\n",
-    "    im_kwargs=dict(cfg.IM_KWARGS),\n",
-    "    marker_titles=markers,\n",
-    "    marker_cmap=\"Set1\",\n",
-    "    marker_kwargs=dict(cfg.MARKER_KWARGS),\n",
-    "    n_cols=10\n",
-    ").show()"
+    "# plot the first phase of images with landmarks\n",
+    "\n",
+    "n_img_per_fig = 35\n",
```

Comment:
If 35 is the default config, specifying so in config.py may be better.

Reply:
No, specifically in this example

Reply:
Why not configure it? so users can just modify the config file?